shopware/core/6.4/public/index.php

```diff
@@ -50,6 +51,13 @@
 
     $shopwareHttpKernel = new HttpKernel($appEnv, $debug, $classLoader);
 
+    $composerPluginLoader = (bool)($context['COMPOSER_PLUGIN_LOADER'] ?? false);
```

Comment:
Can you use the [EnvironmentHelper](https://github.com/shopware/platform/blob/13fc89cadd53c677ccbfb43863175aaa4474642f/src/Core/DevOps/Environment/EnvironmentHelper.php#L23) class?

Reply:
Sure 👍🏻